pycode/memilio-generation/memilio/tools/example_oseir.py

```diff
@@ -41,6 +41,22 @@ def run_memilio_generation(print_ast=False):
     with importlib_resources.as_file(pkg.joinpath('../tools/config.json')) as path:
         with open(path) as file:
             conf = ScannerConfig.schema().loads(file.read(), many=True)[0]
+
+    home_dir = os.path.expanduser("~")
```

Comment:
Maybe just use `file_path = os.path.dirname(os.path.abspath(__file__))` (the tools folder, aka the folder of this file) as the target and point to source file with relative path beginning at `file_path`. Because its not dependent on user specific file structure outside of the package. This seems to expect the package to be in the home folder and cloned in a folder memilio.